dart/lib/src/enricher/io_enricher_event_processor.dart

```diff
@@ -83,6 +84,7 @@ class IoEnricherEventProcessor extends EventProcessor {
     return <String, dynamic>{
       if (packageConfig != null) 'package_config': packageConfig,
       'number_of_processors': Platform.numberOfProcessors,
+      'isolate': Isolate.current.debugName,
```

Comment:
worth testing on Flutter/Dart Web, since Isolates have a few limitations on the web

Reply:
Well this is the IO processor. It is not used on the web?
But could check if we can test the same on Flutter.

Reply:
Don't see how to add this to any of the flutter tests.
But `Isolate.current.debugName` works in Flutter, I use it heavily and currently have a custom processor which adds this as well.

Isolates are not supported on Web target.

Reply:
Yep, this is IO only just wanted to check, all good then

Reply:
It's nullable, so we should just add it, in case it's not null
```suggestion
      if(Isolate.current.debugName != null)
      'isolate': Isolate.current.debugName,
```

Reply:
From what I have seen, it is never `null`, as it will default to the entry point name.

```dart
void foo(dynamic message) {
  print(Isolate.current.debugName);
  print(message);
}

void main() {
  Isolate.spawn(
    foo,
    'yay',
  );
}
```
```
[+1752 ms] flutter: foo
[        ] flutter: yay
```

That's why the main Flutter isolate is called `main`.

I would actually default it to something in case it actually is `null`.
Something like `Isolate.current.debugName ?? 'unknown'`?

Reply:
Huh, interesting TIL. I would say, not adding it is better than setting it to `unknown` because you could actually set it to `unknown`. I don't believe anybody would actually do that, but you'll never know.

```dart
void foo(dynamic message) {
  print(Isolate.current.debugName);
  print(message);
}

void main() {
  Isolate.spawn(
    foo,
    'unknown',
  );
}
```